test/quantization/test_numeric_suite_fx.py

```diff
@@ -231,3 +232,86 @@ def compare_and_validate_results(float_model, q_model, module_swap_list, data):
         compare_and_validate_results(
             backup_prepared_model, q_model, module_swap_list, linear_data
         )
+
```

Comment:
At a higher level can we merge the tests for numeric_suite and fx into a single file? The only difference in the tests would be that in one case, we quantize with fx APIs and call the fx numeric suite APIs. In the other case, we call with eager mode APIs.  This way, we can share the exact same models and reuse most of the testing code.

Reply:
I would prefer keep them separate at this time, especially when we are still refining the APIs.  Fx and non-fx test cases use slightly different model definition. We can always merge them down the road.